src/pages/stepsCalculator/stepsCalculator.tsx

```diff
@@ -0,0 +1,148 @@
+import React, { useState } from 'react';
+import {
+    IonButton,
+    IonCol,
+    IonContent,
+    IonHeader,
+    IonInput,
+    IonItem,
+    IonLabel,
+    IonNote,
+    IonRouterLink,
+    IonTitle,
+    IonToolbar,
+    IonGrid,
+    IonRow,
+} from '@ionic/react';
+//import ExploreContainer from '../../components/ExploreContainer';
```

Comment:
Do we still need this commented out code?

Reply:
Nope, will be removed